index.html

```diff
@@ -2,12 +2,189 @@
+              Black coffee is LOVE
+              <i class="fa fa-heartbeat" aria-hidden="true"></i>
+            </dd>
+            <br /><br />
+            <dt><i class="fa fa-bicycle" aria-hidden="true"></i> Hobbies:</dt>
+            <br />
+            <dd>Netflix K-dramas, movies</dd>
+            <dd>Computer (steam) RTS games</dd>
+            <dd>Jogging, cycling</dd>
+          </dl>
+        </div>
+        <div>
+          <dl>
+            <dt><i class="fa-solid fa-code"></i> Programming skills:</dt>
+            <br />
+            <dd class="text-info">- HTML</dd>
+            <dd class="text-info">- CSS</dd>
+            <dd class="text-info">- Javascript</dd>
+            <dd class="text-info">- Python</dd>
+            <dd class="text-info">- C++ (Arduino)</dd>
+            <br />
+            <dt>
+              <i class="fa-solid fa-screwdriver-wrench"></i> Frameworks / Tools:
+            </dt>
+            <br />
+            <dd class="text-info">- jQuery</dd>
+            <dd class="text-info">- Bootstrap</dd>
+            <dd class="text-info">- React / React Native</dd>
+          </dl>
+        </div>
+        <div>
+          <dl>
+            <dt><i class="fa-solid fa-graduation-cap"></i> Education:</dt>
+            <br />
+            <dd>National University of Singapore</dd>
+            <br />
+            <dd class="text-info">- Bachelor of Science (Nursing)</dd>
+            <br /><br />
+            <dd>Singapore Polytechnic PACE</dd>
+            <br />
+            <dd class="text-info">- Diploma (Conversion) Web & Programming</dd>
+            <dd class="text-info">- Full Stack Web Development</dd>
+            <br /><br />
+            <dd>Institute of Adult Learning</dd>
+            <br />
+            <dd class="text-info">
+              - Advanced Certificate in Learning & Performance
+            </dd>
+          </dl>
+        </div>
+        <div>
+          <dl>
+            <dt><i class="fa-solid fa-calendar-days"></i> Employment:</dt>
+            <br />
+            <dt>2014 - 2015</dt>
+            <br />
+            <dd>National University Hospital</dd>
+            <br />
+            <dd class="text-info">- Staff Nurse (Cardiology)</dd>
+            <br /><br />
+            <dt>2015 - 2022</dt>
+            <br />
+            <dd>Nullspace Robotics</dd>
+            <br />
+            <dd class="text-info">- Part-time / Full-time Trainer</dd>
+            <dd class="text-info">- Senior Trainer</dd>
+            <dd class="text-info">- Senior Manager, Training & Curriculum</dd>
+            <dd class="text-info">- Head of Technology</dd>
+          </dl>
+        </div>
+      </div>
+    </div>
+
+    <!-- Portfolio -->
+    <div id="portfolio">
+      <h1>My Projects</h1>
+      <div class="portfolio-grid">
+        <div class="portfolio-card">
+          <a href="http://tcpc.org.sg/"
+            ><img src="images/church.jpeg" alt="church photo"
+          /></a>
+          <br />
```

Comment:
here it could have made sense to use margins/paddings to create the space between elements